src/redux/actions/user/index.js

```diff
@@ -51,3 +53,19 @@ export const fetchUserData = username => (dispatch) => {
 export const clearDictionaryData = () => (dispatch) => {
   dispatch({ type: CLEAR_DICTIONARY, payload: [] });
 };
+
+export const fetchMemberStatus = url => async (dispatch) => {
+  let response;
+  try {
+    response = await instance.get(url);
+  } catch (error) {
+    if (error.response.status === 403 || error.response.status === 404) {
+      dispatch(isErrored(false, USER_IS_NOT_MEMBER));
+    }
+    return dispatch(isFetching(false));
```

Comment:
I think it makes sense to stop the loader then take the success action instead of the reverse unless there is a good reason you must take the success action while the loader still loads. Same goes for the error. Stop the loader then show the error. Instead of showing an error then stop the loader.

Reply:
I have made the update.